app/views/welcome/login.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="row">
```

Comment:
This file seems to be lacking a bit of correct formatting (e.g. indentations), doesn't it?